modules/sdk-core/src/account-lib/mpc/curves/secp256k1.ts

```diff
@@ -4,6 +4,9 @@ import * as secp from '@noble/secp256k1';
 
 const order = secp.CURVE.n;
 
+/**
+ * @deprecated use Secp256k1Curve from sdk-lib-mpc instead
+ */
 export class Secp256k1Curve implements BaseCurve {
```

Comment:
Why can't we also export the Secp256k1Curve from sdk-lib-mpc along side the deprecation notice?, since making changes in the Secp256k1Curve should also be reflected here until we remove it from here

Reply:
Good idea, didn't think about re-exporting from the deprecated file.

Reply:
Fixed.